include/deal.II/base/mpi.h

```diff
@@ -719,6 +719,21 @@ namespace Utilities
     MinMaxAvg
     min_max_avg(const double my_value, const MPI_Comm &mpi_communicator);
 
+    /**
+     * Same as above but returning the sum, average, minimum, maximum,
+     * process id of minimum and maximum as a collective operation of on the
+     * given MPI @ref GlossMPICommunicator "communicator" @p mpi_communicator
+     * for each entry of the vector.
+     *
+     * @note This function performs a single reduction sweep.
+     *
+     * @req Size of the input vector has to be the same on all processes.
+     */
+    std::vector<MinMaxAvg>
+    min_max_avg(const std::vector<double> my_values,
```

Comment:
clang-tidy rightfully complains that you pass the vector by value rather than reference.